docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -1073,18 +1128,21 @@ quarkus.oidc.client-id=quarkus-app
 quarkus.oidc.token.audience=${quarkus.oidc.client-id}
 ----
 
-If you set `quarkus.oidc.client-id` but your endpoint does not require remote access to one of OpenID Connect Provider endpoints (introspection, token acquisition, etc.) then do not set a client secret with the `quarkus.oidc.credentials` or similar properties as it will not be used.
+If you set `quarkus.oidc.client-id`, but your endpoint does not require remote access to one of the OIDC provider endpoints (introspection, token acquisition, and so on), do not set a client secret with `quarkus.oidc.credentials` or similar properties because it will not be used.
 
-Note Quarkus `web-app` applications always require `quarkus.oidc.client-id` property.
+[NOTE]
+====
+Quarkus `web-app` applications always require the `quarkus.oidc.client-id` property.
+====
 
 == Authentication after HTTP request has completed
```

Comment:
Not sure if an article is needed here. If so, propose to change to "a" instead of "the"
@sberyozkin: Would the following update be correct for this title: "Authentication after a HTTP request has completed"